.devcontainer/devcontainer.json

```diff
@@ -50,5 +50,5 @@
   },
   "image": "mcr.microsoft.com/devcontainers/python:3",
   "name": "Asynchronous Python client for Twente Milieu API",
-  "updateContentCommand": ". ${NVM_DIR}/nvm.sh && nvm install && nvm use && npm install && poetry install && poetry run pre-commit install"
+  "updateContentCommand": "git config --global --add safe.directory '*' && . ${NVM_DIR}/nvm.sh && nvm install && nvm use && npm install && poetry install && poetry run pre-commit install"
```

Comment:
This change seems unrelated to the actual PR goal/description.

Reply:
This is a change I needed to get up to speed with the dev container you added. I can undo the commit if you'd prefer.